root/programs/C#/Tests/Public.Test/Public.Test.csproj.user

```diff
@@ -0,0 +1,7 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'Debug|AnyCPU'">
+    <StartAction>Program</StartAction>
+    <StartProgram>C:\root\programs\C#\Tests\packages\NUnit.Runners.2.6.3\tools\nunit.exe</StartProgram>
```

Comment:
@daisukenishino 

>What is the runtests.bat.

see:
https://github.com/OpenTouryoProject/OpenTouryo/blob/fix_test_projects/root/programs/C%23/Tests/Public.Test/bat/runtests.bat

>What thereby becomes convenient in the debug of the test ?

In production environment, Jenkins will execute nunit-console.exe, not nunit.exe.
So, you can test in environment that is close to production environment.